packages/jsx/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@vue/jsx",
```

Comment:
I hope it's okay to use `@vue/jsx` as package name.

Reply:
It's more like `@vue/tsx` though 😂
Unless we are planning to add more functionalities to this package later.